src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,25 +136,14 @@ private void configureButtonBindings() {
     }, m_revolver));
 
     m_hood.setDefaultCommand(new ManualHood(() -> {
-      if (m_operatorJoystick.getPOV() == 0 || m_operatorJoystick.getPOV() == 45 || m_operatorJoystick.getPOV() == 315) {
-        return 0.6;
-      } else if (m_operatorJoystick.getPOV() == 180 || m_operatorJoystick.getPOV() == 135
-          || m_operatorJoystick.getPOV() == 225) {
-        return -0.6;
-      }
-      return 0.0;
+      return Utilities.deadzone(m_operatorJoystick.getRawAxis(RobotMap.JOYSTICK_AXIS.HOOD));
```

Comment:
Do we still need the `ManualHood` command, or should this just be a RunCommand? If we leave this in here, we should remove `m_hood.enable()` from the `ManualHood` command.

Reply:
I'm not sure I follow the reasoning here, why can't we have `m_hood.enable()` in the `end`?